contracts/Pricer.sol

```diff
@@ -357,9 +371,9 @@ contract Pricer is OpsManaged, PricerInterface {
         view
         returns (uint256, uint256) /* number of BT ,number of commission BT */
     {
-        uint256 adjConversionRate = SafeMath.mul(pricerConversionRate, 10**uint256(pricerDecimals));
-        uint256 amountBT = SafeMath.div(SafeMath.mul(_transferAmount, adjConversionRate), _pricePoint);
-        uint256 commissionAmountBT = SafeMath.div(SafeMath.mul(_commissionAmount, adjConversionRate), _pricePoint);
+        uint256 adjConversionRate = pricerConversionRate.mul(10**uint256(pricerDecimals)).div(10**uint256(pricerConversionRateDecimals));
+        uint256 amountBT = _transferAmount.mul(adjConversionRate).div(_pricePoint);
```

Comment:
What is the order of operation ? will it multiply first as multiplication and division has same precedence.

Reply:
@deepesh-kn, think of it less as math operations and more as standard method chaining; so it calls `mul` first and then calls `div` on the return from `mul`. If `div` came first, then it would divide first.

Reply:
ahhh !!!! totally makes sense :)